recipes/ujson/meta.yaml

```diff
@@ -0,0 +1,47 @@
+package:
+  name: ujson
+  version: 1.35
+
+source:
+  git_url: https://github.com/esnme/ultrajson.git
```

Comment:
There's a `sha256sum` command line utility in Windows and Linux that can be used to calculate the SHA256 hash.